dbusmock/testcase.py

```diff
@@ -251,3 +278,30 @@ def spawn_server_template(cls, template: str, parameters: Dict[str, Any] = None,
                         dbus_interface=MOCK_IFACE)
 
         return (daemon, obj)
+
+    @classmethod
+    def enable_service(cls, service, system_bus: bool = False) -> bool:
+        '''Enable the given well known service name inside dbusmock
+
+        This symlinks a service file from the usual dbus service directories
+        into the dbusmock environment. Doing that allows the service to be
+        launched automatically.
+
+        Note that this function only checks the paths in $XDG_DATA_DIRS.
+
+        Returns True if a service file was found and symlinked.
+        '''
+        if system_bus:
+            services_dir = 'system-services'
+        else:
+            services_dir = 'services'
+        xdg_data_dirs = '/usr/local/share/:/usr/share/'
+        if 'XDG_DATA_DIRS' in os.environ and os.environ['XDG_DATA_DIRS']:
+            xdg_data_dirs = os.environ['XDG_DATA_DIRS']
```

Comment:
Please write this as:

   xdg_data_dirs = os.environ.get('XDG_DATA_DIRS', '/usr/local/share/:/usr/share/')

Reply:
Unfortunately, that wouldn't conform to the XDG specification. i.e. an empty XDG_DATA_DIRS also needs to be handled as the default value.

Reply:
OK, then

    os.environ.get('XDG_DATA_DIRS') or '/usr/local/share/:/usr/share/'

?